modules/openapi-generator/src/main/resources/php-symfony/services.mustache

```diff
@@ -24,6 +24,7 @@ services:
 {{#operations}}
     {{bundleAlias}}.controller.{{pathPrefix}}:
         class: {{controllerPackage}}\{{baseName}}Controller
+        public: true
```

Comment:
Tagging the controllers with `controller.service_arguments` is the superior solution since it's a tag tailor-made for that use case. This solution goes about it from the wrong side.